vmc/utils.go

```diff
@@ -71,3 +74,17 @@ func expandMsftLicenseConfig(config []interface{}) *model.MsftLicensingConfig {
 	licenseConfig = model.MsftLicensingConfig{MssqlLicensing: &mssqlLicensing, WindowsLicensing: &windowsLicensing}
 	return &licenseConfig
 }
+
+func getNSXTReverseProxyURLConnector(nsxtReverseProxyUrl string) (client.Connector, error) {
+	apiToken := os.Getenv(APIToken)
```

Comment:
What if env is not set?

Reply:
Token is validated and set into env variable when terraform is first initialized - https://github.com/vmware/terraform-provider-vmc/blob/master/vmc/provider.go#L81. Terraform apply will error out if it's not set.